packages/opentelemetry-metrics/src/BoundInstrument.ts

```diff
@@ -134,8 +133,7 @@ export class BoundMeasure extends BaseBoundInstrument
 /**
  * BoundObserver is an implementation of the {@link BoundObserver} interface.
  */
-export class BoundObserver extends BaseBoundInstrument
-  implements types.BoundObserver {
+export class BoundObserver extends BaseBoundInstrument {
```

Comment:
Shouldn't there be no bound observer?

Reply:
> https://github.com/open-telemetry/oteps/blob/master/text/0072-metric-observer.md#observer-calling-conventions
> A bound calling convention is not needed for Observer instruments because there is little if any performance benefit in doing so (as Observer instruments are called during collection, there is no need to maintain "active" records concurrent with collection).

It's not expected to be a part of public API, i.e. there is no `Observer.bind`. However, we do need concrete label value bound observers, which are created implicitly on `ObserverResult.observe`.